src/irmin-pack-utils/ppcf.ml

```diff
@@ -0,0 +1,30 @@
+module Io = Irmin_pack_unix.Io.Unix
+module Io_errors = Irmin_pack_unix.Io_errors.Make (Io)
+module Control = Irmin_pack_unix.Control_file.Upper (Io)
+
+let main control_file =
+  Fmt_tty.setup_std_outputs ~style_renderer:`Ansi_tty ();
+  let r = Control.read_raw_payload ~path:control_file in
+  match r with
+  | Error err -> Io_errors.raise_error err
+  | Ok payload ->
+      Fmt.pr "%a\n"
+        (Repr.pp_json Irmin_pack_unix.Control_file.Payload.Upper.raw_payload_t)
+        payload
+
+(** Cmdliner **)
+
+open Cmdliner
+
+let control_file =
+  Arg.(
+    required
+    & pos 0 (some string) None
+    & info [] ~docv:"control file" ~doc:"the path to the control file")
+
+let main_cmd =
+  let doc = "a visual tool for irmin pack files inspection" in
```

Comment:
Unless this is intended to have other functionality.
```suggestion
  let doc = "a json printer for irmin pack control files" in
```

Reply:
No, just a dumb copy&paste&forget_to_change_it, thx :D